Sources/SwiftParser/Lookahead.swift

```diff
@@ -274,6 +274,13 @@ extension Parser.Lookahead {
     var lookahead = self.lookahead()
     lookahead.eat(.leftBrace)
 
+    // '@_accessorBlock' is a builtin disambiguation marker.
+    if lookahead.peek(isAt: .identifier),
```

Comment:
Don’t you also need to consume the `@` before looking at the identifier? Also, I think a test case would be good.

Also, I think this would probably belong in the ` while lookahead.consume(if: .atSign) != nil {` loop below to make it resilient so `@_accessorBlock` doesn’t necessarily need to be the first attribute.